tasks/internal_reconcile_report_orphan/test/unit_tests/adapters/storage/test_rdbms.py

```diff
@@ -55,9 +53,6 @@ def test_get_orphans_page_happy_path(
 
         storage_adapter = rdbms.StorageAdapterRDBMS(connection_uri)
 
-        mock_make_url.assert_called_once_with(connection_uri)
-        mock_create_engine.assert_called_once_with(mock_make_url.return_value, future=True)
-
```

Comment:
You still need the mock_create_engine check, it'll just be tweaked to `mock_create_engine.assert_called_once_with(connection_uri, future=True)`

Reply:
added